__mock__/add-remove.test.js

```diff
@@ -0,0 +1,34 @@
+describe('check Add Remove function', () => {
+  // 1-Arrange
+/* eslint-disable */
```

Comment:
- [ ] Kindly remove this eslint disable line, it's better to fix the issue in the code.